biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/CommandLineResources.scala

```diff
@@ -22,6 +22,8 @@ import org.broadinstitute.gatk.queue.function.CommandLineFunction
  */
 trait CommandLineResources extends CommandLineFunction with Configurable {
 
+  lazy val maxThreads: Option[Int] = config("max_threads")
+
   def defaultThreads = 1
```

Comment:
what is config("threads") and config("max_threads")? would be better if it was explained here with code comments